source/containers/Form/hooks/useForm.ts

```diff
@@ -132,7 +157,6 @@ function useForm(initialState: FormReducerState) {
    */
   const handleBlur = (answer: Record<string, any>, questionId: string) => {
     dispatch({ type: 'DIRTY_FIELD', payload: { answer, id: questionId } });
-    dispatch({ type: 'VALIDATE_ANSWER', payload: { answer, id: questionId, checkIfDirty: true } });
```

Comment:
Hmm, are you sure this should be removed? This leads to no validation the first time you blur an input field.